modules/administration-guide/nav.adoc

```diff
@@ -70,6 +70,7 @@
 **** xref:configuring-oauth-1-for-a-bitbucket-server.adoc[]
 **** xref:configuring-oauth-2-for-microsoft-azure-devops-services.adoc[]
 *** xref:removing-user-data-in-compliance-with-the-gdpr.adoc[]
+*** xref:configuring-user-cluster-roles.adoc[]
```

Comment:
I recommend swapping the order of these two sections because user roles is likely to be more important than removing user data and because removing user data can be locally the last of these sections:
```suggestion
*** xref:configuring-user-cluster-roles.adoc[]
*** xref:removing-user-data-in-compliance-with-the-gdpr.adoc[]
```